src/workerd/io/worker-fs.c++

```diff
@@ -1632,12 +1632,42 @@ class DevRandomFile final: public File, public kj::EnableAddRefToThis<DevRandomF
   mutable kj::Maybe<kj::String> maybeUniqueId;
 };
 
+// Write stdio via console.log. Somewhat convoluted, but this then supports:
+// - inspector reporting
+// - structured logging
+// - stdio output otherwise
+void writeStdio(jsg::Lock& js, VirtualFileSystem::Stdio type, kj::ArrayPtr<const kj::byte> bytes) {
+  auto chars = bytes.asChars();
+  size_t endPos = chars.size();
+  if (chars[endPos - 1] == '\n') endPos--;
+  auto context = js.v8Context();
```

Comment:
You can use `js.global()` here to get the `Global` as a `jsg::JsObject` that'll be a bit nicer to use in here.